Aspire Budgeting/Views/TabBarView/ProminentTabBarItemView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ProminentTabBarItemView.swift
+//  Aspire Budgeting
+//
+
+import SwiftUI
+
+struct ProminentTabBarItemView: View {
+  
+  var width: CGFloat = 70
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/845606672c945b4a26becac88e482112dc55b51f/Aspire Budgeting/Views/TabBarView/ProminentTabBarItemView.swift#L11"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>ProminentTabBarItemView.swift:11</code></td>
    </tr>
  </tbody>
</table>